server/services/locationService.ts

```diff
@@ -0,0 +1,31 @@
+import { Prison } from '../types/uof'
+
+import logger = require('../../log.js')
+
+import incidentClient = require('../data/incidentClient')
```

Comment:
this is a dependency so should be injected rather than required.

Reply:
done